shared/src/abis/Permit2.ts

```diff
@@ -0,0 +1,901 @@
+export const Permit2ABI = [
```

Comment:
Two things:
- Did you delete this from `earn/src`? git isn't recognizing it as a moved file.
- Let's stick with `permit2ABI` to match wagmi's capitalization style

Reply:
Will delete, and I am cool with doing this.